WeakAuras/WeakAuras.lua

```diff
@@ -2850,6 +2850,24 @@ function WeakAuras.Modernize(data)
     end
   end
 
+  -- Version 14 was introduced March 2019 in BFA
+  if data.internalVersion < 14 then
+    if data.triggers then
+      for triggerId, triggerData in pairs(data.triggers) do
+        if type(triggerData) == "table"
+        and triggerData.trigger
+        and triggerData.trigger.debuffClass
+        and type(triggerData.trigger.debuffClass) == "string"
+        and triggerData.trigger.debuffClass ~= ""
+        then
+          local idx = trigger.debuffClass..""
+          data.triggers[triggerId].trigger.debuffClass = {}
+          data.triggers[triggerId].trigger.debuffClass[idx] = true
```

Comment:
You should be able to simplify these two statements to `triggerData.trigger.debuffClass = { [idx] = true }`

Reply:
Also what’s with the concatenation with an empty string?

Reply:
will fix